### ISSUE TITLE
Test failures on main

### ISSUE THREAD

**Issue by Robbybp** [lanl-ansi/PowerModelsRestoration.jl#76]
See e.g. https://github.com/lanl-ansi/PowerModelsRestoration.jl/actions/runs/11598737054/job/32295250780

I can only reproduce the test-ac-ml-uc-heuristic failure locally. We seem to be hitting an empty array in `PowerModels.calc_connected_components`.

**Comment by noahrhodes** [comment]
I assume this update changed something that results in hitting an empty array
https://github.com/lanl-ansi/PowerModels.jl/pull/914

**Comment by Robbybp** [comment]
Yeah, that's likely the culprit. Have submitted a patch upstream https://github.com/lanl-ansi/PowerModels.jl/pull/933

**Comment by noahrhodes** [comment]
I suspect it just needs to be relaxed.  I assume objective value is still identical, and gen/load increases in consecutive time steps?
